### ISSUE TITLE
Test other ADO.NET providers

### ISSUE THREAD

**Issue by divega** [aspnet/DataAccessPerformance#7]

**Comment by bgrainger** [comment]
I volunteer to help with https://github.com/mysql-net/MySqlConnector

**Comment by sebastienros** [comment]
@bgrainger we already have the numbers for this one.

**Comment by sebastienros** [comment]
@bgrainger Are you in touch with @divega already? We need your help to improve these numbers.

**Comment by bgrainger** [comment]
> we already have the numbers for this one.

Yep, I just saw this line: https://github.com/aspnet/DataAccessPerformance/blob/master/src/BenchmarkDb/BenchmarkDb.csproj#L15

> Are you in touch with @divega already?

We had an email conversation about six weeks ago; haven't communicated since then.

**Comment by divega** [comment]
@bgrainger good to hear from you here. I will create a new issue specific to MySqlConnector and we can discuss there how to get started.

**Comment by divega** [comment]
Created https://github.com/aspnet/DataAccessPerformance/issues/21 for MySqlConnector.

**Comment by cincuranet** [comment]
I created PR #44 for Firebird.
